components/inputs/input-time.js

```diff
@@ -36,6 +35,18 @@ function getIntervalNumber(size) {
 	}
 }
 
+function getDefaultTime(time) {
+	switch (time) {
+		case 'endOfDay':
+			return END_OF_DAY;
+		case 'startOfDay':
+		case undefined:
+			return new Date(TODAY.year, TODAY.month, TODAY.date, 0, 0, 0);
+		default:
```

Comment:
What happens if it fails to parse here? I guess `parseValue` which calls `parseISOTime` throws an exception, which is the same as what would happen with an invalid value?

Reply:
When `parseValue` is given something invalid it returns a date object with a time equivalent to `00:00:00`, `parseTime` is the one that returns null which we use with invalid input values, if I understand what you're referencing.

Reply:
Updated this so that it will throw.